cpp/include/IceUtil/Monitor.h

```diff
@@ -8,6 +8,7 @@
 #include <IceUtil/Config.h>
 #include <IceUtil/Lock.h>
 #include <IceUtil/Cond.h>
+#include <IceUtil/Mutex.h>
```

Comment:
Do we still use Mutex and Monitor? If not, I would remove them entirely.

Reply:
I still have to fix the tests, Application and Timer classes.